app/components/viral/attachment_label_component.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Viral
+  # Viral component for attachments table colored labels
+  class AttachmentLabelComponent < Viral::Component
+    attr_reader :label
+
+    COLORS = {
+      format: {
+        fasta: 'bg-blue-100 text-blue-800 text-base font-medium
```

Comment:
Okay, my bad. I had thought the font size was a little small, hence the change. I've changed to the pill badge classes. And I assume you meant change the pill to a `span` rather than the `div` that was already there?

Fixed in 4f9fbc4c1f178c4e6fa1e5f394e88f54b37535c2